src/openzaak/components/documenten/tests/admin/test_eiocanonical_inline_audittrail.py

```diff
@@ -136,3 +137,36 @@ def test_eio_add(self):
 
         new_data = audittrail.nieuw
         self.assertEqual(new_data["identificatie"], "12345")
+
+    def test_eio_add_no_version(self):
```

Comment:
This test file is specifically for audittrail stuff, so I don't think this is the right place for this test.